src/main/java/com/mraof/minestuck/alchemy/CardCaptchas.java

```diff
@@ -49,7 +48,16 @@ public static String getCaptcha(Item item, MinecraftServer server)
 		MSExtraData data = MSExtraData.get(server);
 		CardCaptchas captchas = data.getCardCaptchas();
 		
-		if(captchas.captchasMap.containsKey(item))
+		if(PredeterminedCardCaptchas.predefinedCardMap.containsKey(item) && captchas.captchasMap.containsKey(item))
+		{
+			captchas.captchasMap.remove(item);
```

Comment:
Remember to call `data.setDirty();` whenever making changes to generated captchas. (also in `getItemFromCaptcha()`)